cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -89,7 +89,7 @@ func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 			// since the binary config file jaeger_query's ui.config_file points to
 			// "./cmd/jaeger/config-ui.json"
 			Dir: "../../../..",
-			Env: envVars,
+			Env: append(os.Environ(), envVars...),
```

Comment:
I am not sure we want such a blanket copy, I would rather use some kind of whitelist of env vars that should be propagated, and have that list controlled by the individual tests (i.e. empty by default)

Reply:
@yurishkuro Addressed. Let me know what you think